native/src/modules/layout/containers/HeaderContainer.js

```diff
@@ -87,13 +92,20 @@ const mapStateToProps = (state: StateType, ownProps: OwnPropsType): StatePropsTy
       }
     })
   }
+
   const peeking = isPeekingRoute(state, { routeCity: route.city })
-  const shareUrlFromScene: ?string = typeof ownProps.scene.route.params?.shareUrl === 'string' ? ownProps.scene.route.params?.shareUrl : null
-  const { city, language } = route
-  const path = route.path || null
-  const shareUrl = shareUrlFromScene || getShareUrl({ city, language, path, routeName: ownProps.scene.route.name })
+  const language = route.language
+  const path = route.path || undefined
```

Comment:
why can't you just write `const path = route.path` ?

Reply:
@klinzo Its part of you code, isn't it?

Reply:
Possibly :thinking: Not sure if there was a reason for this, but if there is a reason it's because of flow. Just try it and simplify it as suggested if it works.

Reply:
Yeah its Flow.

Cannot get route.path because property path is missing in object type [1].
where [1] is NewsRouteState Type.